build/app_template_files/__NativeSwiftTemplateAppName__/__NativeSwiftTemplateAppName__/SalesforceSDKAssets.xcassets/Contents.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Or to put it another way, the asset catalog should be the responsibility of the SDK, as it is for independent CocoaPods installs for third party apps.